src/ProjectModule/components/ProjectList/ProjectList.tsx

```diff
@@ -13,6 +15,17 @@ interface progectListType {
 }
 
 export default function ProjectModule() {
+
+    // dark Light moda
+    const darkModeContext = useDarkMode();
+
+    if (!darkModeContext) {
```

Comment:
ℹ️ Instead of returning null when dark mode context is not available, you might want to render a loading state or handle the case differently.